pkg/search/result.go

```diff
@@ -82,10 +110,47 @@ type License struct {
 
 type User struct {
 	GravatarID string `json:"gravatar_id"`
-	ID         int64  `json:"id"`
+	ID         string `json:"node_id"`
 	Login      string `json:"login"`
 	SiteAdmin  bool   `json:"site_admin"`
 	Type       string `json:"type"`
+	URL        string `json:"url"`
+}
+
+type Issue struct {
+	Assignees         []User           `json:"assignees"`
+	Author            User             `json:"user"`
+	AuthorAssociation string           `json:"author_association"`
+	Body              string           `json:"body"`
+	ClosedAt          time.Time        `json:"closed_at"`
+	CommentsCount     int              `json:"comments"`
+	CreatedAt         time.Time        `json:"created_at"`
+	HTMLURL           string           `json:"html_url"`
+	ID                string           `json:"node_id"`
+	Labels            []Label          `json:"labels"`
+	IsLocked          bool             `json:"locked"`
+	Number            int              `json:"number"`
+	PullRequestLinks  PullRequestLinks `json:"pull_request"`
+	RepositoryURL     string           `json:"repository_url"`
+	State             string           `json:"state"`
+	Title             string           `json:"title"`
+	URL               string           `json:"url"`
+	UpdatedAt         time.Time        `json:"updated_at"`
+}
+
+type PullRequestLinks struct {
+	DiffURL  string `json:"diff_url"`
+	HTMLURL  string `json:"html_url"`
+	PatchURL string `json:"patch_url"`
```

Comment:
Nit: since we don't seem to be using these fields and it's unlikely that we'll use them, we could just omit them from the Go struct